core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -216,7 +216,8 @@ class LogManager(logDirs: Array[File],
       maxProducerIdExpirationMs = maxPidExpirationMs,
       scheduler = scheduler,
       time = time,
-      brokerTopicStats = brokerTopicStats)
+      brokerTopicStats = brokerTopicStats,
+      logDirFailureChannel = logDirFailureChannel)
```

Comment:
Also, to be clear, I understand that the JBOD PR sort of just followed the existing pattern in that class.